### DISCUSSION TITLE
JSON workflows

### DISCUSSION THREAD

**Discussion by marc211** [elsa-workflows/elsa-core#4539] [Q&A]
When is this “JSON Workflows” available？

![image](https://github.com/elsa-workflows/elsa-core/assets/17902923/6dce009e-081b-44f4-a9d2-f4f6555ec47c)

**Comment by sfmskywalker** [comment]
Hi there! Thanks for reaching out. Currently, I'm deeply focused on Elsa 3, which has been taking up the majority of my time. As the sole maintainer for Elsa, my bandwidth is quite limited. While I would love to provide more comprehensive documentation for Elsa 2, I can't give a definite timeline for when, or even if, I'll be able to address that particular guide. I understand the importance of documentation and will certainly do my best. In the meantime, if there are any specific questions or issues you have, please let me know, and I'll do my best to help!

**Comment by sfmskywalker** [comment]
Should you be using Elsa 3, there’s a guide for using JSON workflows here: https://v3.elsaworkflows.io/docs/guides/loading-workflows-from-json
